### PR TITLE
[Console] Table vertical rendering usage

### LINKED ISSUE #16795: [Console] Table vertical rendering
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#41676
| PR author(s) | @yoannrenard
| Merged in    | 6.1

### PR DESCRIPTION
**Pull request by yoannrenard**
Closes #16795

### CODE REVIEW

File: components/console/helpers/table.rst [hunk 1]
```diff
@@ -150,6 +150,30 @@ The output of this command will be:
     |                (the rest of the rows...)            |
     +-------+------------+--------------------------------+
 
+By default, the table is rendered horizontally, which may not look great if
+the number of columns you want to display or the data they contain is too large.
```
Comment (94noni) [inline components/console/helpers/table.rst]: Too long instead of too large?

File: components/console/helpers/table.rst [hunk 2]
```diff
@@ -150,6 +150,30 @@ The output of this command will be:
...
+
+If you prefer to render your table vertically, use the
```
Comment (94noni) [inline components/console/helpers/table.rst]: I would instead write:

« Console tables content are displayed horizontally by default.
You can change this behavior via the setVertical() method » (and the example: for me the example is more worth it that the full description)

also I think you may need to add a version added directive as of 6.1 :)

Comment (yoannrenard) [reply components/console/helpers/table.rst]: Thanks for your help @94noni

**Comment by javiereguiluz** [comment]
Thank you Yoann!

**Pull request merged**